### PR TITLE
Tests: replace grunt-contrib-qunit with jQuery test runner

### PR DESCRIPTION
**Pull request by timmywil**
- replace grunt-contrib-qunit with new jQuery test runner
- Add node workflow to test ui against different jQuery Core versions in Chrome, FF, Edge, and Safari, as well as run the build on Node 10
- Run Node workflow every week
- add filestash workflow to update git files
- the plan is to drop jenkins and testswarm after the 1.13.3 release
- add retries support

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    strategy:
+      fail-fast: false
+      matrix:
+        BROWSER: [chrome, firefox]
+        JQUERY:
+          - "git"
+          - "3.x-git"
+          - "3.7.1"
+          - "2.2.4"
+          - "1.12.4"
```
[the remark this answers is not in the document]
Comment (timmywil) [reply .github/workflows/node.js.yml:29]: I tried to find a way to do this, but there doesn't seem to be a way to do this in github actions, especially when defining a matrix. GA doesn't support references. The closest thing is environment variables and contexts, none of which support sequences like this. And even if it did, most contexts are not available when defining a matrix.

File: tests/runner/command.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+	} )
+	.option( "concurrency", {
+		alias: "c",
+		type: "number",
+		description: "Run tests in parallel in multiple browsers. Defaults to 8."
+	} )
+	.option( "run-id", {
+		type: "string",
+		description: "A unique identifier for this run."
+	} )
```
[the remark this answers is not in the document]
Comment (timmywil) [reply tests/runner/command.js:63]: Actually, it's not needed for selenium.

**Comment by timmywil** [comment]
Added support for retries and set retries to `3` in the jobs.

File: .github/workflows/node.js.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (mgol) [reply .github/workflows/node.js.yml:30]: OK, if there's no way then I'll just need to remember to update all entries. At least it's all in one file.

**Review by mgol — APPROVED** [review]
Thanks!
